openapi-bal-task-plugin/src/main/java/io/ballerina/openapi/bal/tool/OpenAPICodeGeneratorTool.java

```diff
@@ -230,8 +259,15 @@ public static ImmutablePair<OASClientConfig, OASServiceMetadata> extractOptionDe
                     clientMetaDataBuilder.withResourceMode(value.contains("resource"));
                     break;
                 case LICENSE:
-                    String licenseContent = Objects.equals(value, "") ? null :
-                            getLicenseContent(toolContext, Paths.get(value));
+                    String licenseContent;
+                    if (Objects.equals(value, "")) {
```

Comment:
The `value` variable here is a String. Hence, can't we do like this:
```suggestion
                    if ("".equals(value)) {
```

I would prefer a static constant to represent `""`.